dask/dataframe/multi.py

```diff
@@ -493,6 +493,12 @@ def merge(
         left_on = right_on = on
         on = None
 
+    supported_how = ("left", "right", "outer", "inner")
+    if how not in supported_how:
+        raise Exception(
```

Comment:
Let's raise a more specific error here instead of the `Exception` base class. A `ValueError` seems appropriate in this particular case. (Sorry I should have brought this up in my earlier comment)

Reply:
no problem, : ) just updated